app/Http/Controllers/AlbumController.php

```diff
@@ -135,7 +136,7 @@ public function destroy(Request $request, $id)
         $user = $request->user();
         $album = Album::findOrFail($id);
 
-        abort_unless($user->group->is_modo || $user->id === $album->user_id && Carbon::now()->lt($album->created_at->addDay()), 403);
+        abort_unless($user->group->is_modo || ($user->id === $album->user_id && Carbon::now()->lt($album->created_at->addDay())), Response::HTTP_FORBIDDEN);
```

Comment:
Why are we using `Response::HTTP_FORBIDDEN` here over `403`.

Reply:
I think it's easier to read the code, I can return it to 403.

Reply:
`403` is used commonly through out the codebase so it preferred. Unless going to refactor all usages.

Reply:
Ok, reverted to number.